creator-node/stage.env

```diff
@@ -36,6 +36,7 @@ maxRecurringRequestSyncJobConcurrency=50
 snapbackUsersPerJob=2000
 otelTracingEnabled=true
 otelCollectorUrl=https://opentelemetry-collector.staging.audius.co/v1/traces
+mergePrimaryAndSecondaryEnabled=true
```

Comment:
do we not care about the other values for stage? just defaults?

Reply:
correct, the defaults are already [25](https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/config.js#L559) & [50](https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/config.js#L553) which are fine for staging for now - i am overriding these values on foundation nodes back to 25 and 50 but want to lower for SPs for now